kadmin/kadmin.go

```diff
@@ -25,6 +26,48 @@ type Kadmin interface {
 	ConfigUpdater
 	TopicConfigLister
 	SraSetter
+	ClusterConfigLister
+	BrokerConfigLister
+}
+
+type ClusterConfigLister interface {
+	GetClusterConfig() (ClusterConfig, error)
```

Comment:
This should return a `tea.Msg` so the `Update` methods can handle it.
The actual type returned would be something like `ClusterConfigListingStartedMsg` that would contain two `chan`s one for errors and one with the actual result.